python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/abstract_interp_lib_gen.py

```diff
@@ -79,27 +83,6 @@ def aten〇exp〡shape(self: List[int]) -> List[int]:
 def aten〇expm1〡shape(self: List[int]) -> List[int]:
     return upstream_shape_functions.unary(self)
 
-@check_dtype_function([
-    Invocation(NonZeroDTensorWithDtype(torch.float32)),
-    Invocation(NonZeroDTensorWithDtype(torch.float64)),
-    Invocation(NonZeroDTensorWithDtype(torch.bfloat16)),
-    Invocation(NonZeroDTensorWithDtype(torch.int64)),
-    Invocation(NonZeroDTensorWithDtype(torch.int32)),
-    Invocation(NonZeroDTensorWithDtype(torch.bool)),
-    Invocation(ZeroDTensorWithDtype(torch.float32)),
-    Invocation(ZeroDTensorWithDtype(torch.float64)),
-    Invocation(ZeroDTensorWithDtype(torch.bfloat16)),
-    Invocation(ZeroDTensorWithDtype(torch.int64)),
-    Invocation(ZeroDTensorWithDtype(torch.int32)),
-    Invocation(ZeroDTensorWithDtype(torch.bool)),
-])
-def aten〇expm1〡dtype(self_rank_dtype: Tuple[int, int]) -> int:
```

Comment:
Removed dtype functions are just shifted down to be grouped with all the other dtype functions